docs/getting_started/build_an_app.md

```diff
@@ -0,0 +1,378 @@
+# Build an App
+This guide will guide you through the steps of how to build a Lumen app that will look like this:
+
+![](../_static/getting_started/build_app_07.png)
+
+
+
+
+## Building the app
+This section will walk through the steps to creating the app.
+Create a file named `penguins.yaml`.
+The first thing which is needed is a source. Here we will add a penguin source from a file that contains a penguin's table.
```

Comment:
```suggestion
The first thing which is needed is a source. Here we will add file source that will fetch data about individual penguins of three different and various measurement about them (such as the length of their bill).
```

Reply:
I will commit your suggestion, just want to note that the way I built up the sentences was to try to match the words in the sentence with what is happening in the YAML specification.